### PR TITLE
refactor duplicated spreadsheet_list.html 

### LINKED ISSUE #1477: One spreadsheet list to rule them all
We have 2 spreadsheet lists.

https://github.com/openhealthcare/opal/blob/v0.10.0/opal/templates/patient_lists/spreadsheet_list.html
and
https://github.com/openhealthcare/opal/blob/v0.10.0/opal/templates/patient_lists/layouts/spreadsheet_list.html

we would like 1.

Ideally this will have 
- [x] will be the layouts/spreadsheet_list.html (ie we should be able to delete the other one)
- [x] the episode actions block that @pacharanero put into v0.10.0
- [x] make sure we don't lose anything, i.e blocks (inc the episode_actions block introduced by @pacharanero )
- [x] change the patient lists to default to the layouts
- [x] update the change log to state this change

### PR DESCRIPTION
**Pull request by pacharanero**
Closes #1477 

This is also work in progress towards fixing https://github.com/openhealthcare/elcid/issues/1509 via #1457 

Should also close https://github.com/openhealthcare/opal/issues/1513 because there was already a fix in the spreadsheet_view_base.html (which wasn't being used). It is now. I haven't yet tested this but if anyone has an Opal app with just one patient list, a confirmed fix here would enable us to close.

### CODE REVIEW

File: opal/templates/patient_lists/layouts/spreadsheet_list_base.html [hunk 1]
```diff
@@ -3,27 +3,109 @@
 
 <div class="container-fluid content-offset">
 
+  {% block heading %}
```
Comment (davidmiller) [inline opal/templates/patient_lists/layouts/spreadsheet_list_base.html]: I see what you've done here - this is what is in the current list..

We might prefer to have naming convention consistency with the other two lists though, and use `  {% block list_actions %}`.

Comment (davidmiller) [reply opal/templates/patient_lists/layouts/spreadsheet_list_base.html]: (that means deleting the `spreadsheet_list_actions` block as well obv.)

**Comment by davidmiller** [comment]
Overall looks great - just a couple of minor tweaks and then we're good to go!

File: opal/templates/layouts/left_panel.html [hunk 2]
```diff
@@ -1,18 +0,0 @@
-<div class="container-fluid content-offset">
-  {% block heading %}
```
[the remark this answers is not in the document]
Comment (pacharanero) [reply opal/templates/layouts/left_panel.html]: Have added to template removal in changelog, and I cannot find any other instances of it being used anywhere else

File: opal/templates/patient_lists/layouts/spreadsheet_list_base.html (hunk 1 above)
Comment (pacharanero) [reply opal/templates/patient_lists/layouts/spreadsheet_list_base.html]: so is `heading` superfluous?
can it be removed,  and change `spreadsheet_list_actions` to `list_actions`?
I'm not 100% clear on the required end state

**Pull request merged**